src/commands.jl

```diff
@@ -218,7 +218,15 @@ function maybe_step_through_wrapper!(@nospecialize(recurse), frame::Frame)
     length(stmts) < 2 && return frame
     last = stmts[end-1]
     isexpr(last, :(=)) && (last = last.args[2])
-    is_kw = isa(scope, Method) && endswith(String(Base.unwrap_unionall(Base.unwrap_unionall(scope.sig).parameters[1]).name.name), "#kw")
+
+    is_kw = false
+    if isa(scope, Method)
+        param1 = Base.unwrap_unionall(Base.unwrap_unionall(scope.sig).parameters[1])
+        if !(param1 isa Union)
```

Comment:
Maybe something like this would be more robust? 😂 
```suggestion
        unwrap1 = Base.unwrap_unionall(scope.sig)
        if isa(unwrap1, DataType)
            param1 = Base.unwrap_unionall(unwrap1.parameters[1])
            if isa(param1, DataType)
```

Reply:
Yeah I was thinking about that, but then again I wasn't sure if this can be anything else :D